source/css/sass/mixins/_typography.scss

```diff
@@ -82,10 +82,10 @@
 @mixin font-variant-position($position) {
   @if $position == sub {
     @include _base_font-variant-position($position);
-    bottom: -0.25em;
+    bottom: -0.25em; // stylelint-disable-line csstools/use-logical
   } @else if $position == super {
     @include _base_font-variant-position($position);
-    top: -0.5em;
+    top: -0.5em; // stylelint-disable-line csstools/use-logical
```

Comment:
These are fallbacks already, so disabling. (Does show that `logical-property()` doesn't handle `inset` yet.)